tests/files/env_config.yml

```diff
@@ -0,0 +1,9 @@
+environments:
+  development:
+    prefix: Dev-Prefix-
+
+  staging:
+    prefix: Stage-Prefix-
+
+  production:
+    prefix: Prod-Prefix-
```

Comment:
Ah, didn't see that environments are just containers for region files. I was thinking of the environments concept quite differently. Will describe below.